assets/i18n/en.toml

```diff
@@ -106,21 +106,21 @@ remaining = "Remaining"
 [main.loadpointSettings]
 title = 'Settings "{0}"'
 vehicle = "Vehicle"
-currents = "Charging"
+currents = "Charging current"
```

Comment:
Could this be given in range per minute (if the voltage and car model is known)?

Reply:
This is a separate discussion that we should have in a dedicated thread. The UI section that belongs to this text is currently an experimental UI that's invisible for most users.